catalog/plugin/catalog-plugin-metacard-validation/src/main/java/ddf/catalog/metacard/validation/MetacardValidityMarkerPlugin.java

```diff
@@ -216,4 +222,15 @@ public void setEnforceWarnings(boolean enforceWarnings) {
     public boolean getEnforceWarnings() {
         return enforceWarnings;
     }
+
+    protected static class ValidationResult {
```

Comment:
This needs to implement `Serializable`.

Conversely, you could adopt a simple tokenization pattern for the validation results where you join the String representations of the two boolean values with a comma, colon, pipe, etc. Then on retrieval you'd split and convert to boolean to get your values.